modules/kubernetes-addons/promtail/locals.tf

```
@@ -0,0 +1,23 @@
+locals {
+  name = "promtail"
+  default_helm_config = {
+    name             = local.name
+    chart            = local.name
+    repository       = "https://grafana.github.io/helm-charts"
+    version          = "6.3.0"
+    namespace        = try(var.helm_config["namespace"], local.name)
```

Since this is getting merged anyways, this is not needed like this
```suggestion
    namespace        = local.name
```